controllers/article-controller.js

```diff
@@ -31,6 +31,19 @@ function getArticles(request, response, next){
   })
 }
 
+function patchArticle(request,response,next){
+   const {article_id} = request.params
+   updateArticleByID(article_id, request.body)
```

Comment:
I would maybe destructure the update object and pass that in place of sending the entire request.body